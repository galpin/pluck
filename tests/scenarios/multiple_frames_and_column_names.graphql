{
  full: rockets @frame {
    name
    engines {
      type
    }
  },
  short: rockets @frame {
    name
    engines {
      type
    }
  }
}
