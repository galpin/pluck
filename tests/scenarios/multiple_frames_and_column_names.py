def get_kwargs():
    return {"column_names": {"full": "full", "short": "short"}}
