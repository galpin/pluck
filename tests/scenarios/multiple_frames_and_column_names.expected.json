{
 "full": [
  {
   "name": "Falcon 1",
   "engines.type": "merlin"
  },
  {
   "name": "Falcon 9",
   "engines.type": "merlin"
  },
  {
   "name": "Falcon Heavy",
   "engines.type": "merlin"
  },
  {
   "name": "Starship",
   "engines.type": "raptor"
  }
 ],
 "short": [
  {
   "name": "Falcon 1",
   "type": "merlin"
  },
  {
   "name": "Falcon 9",
   "type": "merlin"
  },
  {
   "name": "Falcon Heavy",
   "type": "merlin"
  },
  {
   "name": "Starship",
   "type": "raptor"
  }
 ]
}